docs/guides/self_hosting.md

```diff
@@ -0,0 +1,117 @@
+# [Experimental] Self Hosting
+**This guide is currently experimental. Instructions are subject to change as we improve self-hosting capabilities.**
+
+We provide a Helm chart that deploys LLM Engine to an Elastic Kubernetes Cluster (https://aws.amazon.com/eks/). This Helm chart should be configured to connect to dependencies (such as a postgresql database) that you may already have available in your environment.
+
+The only portions of the helm chart that are production ready are the parts that configure and manage LLM Server itself—not Postgres, IAM, etc.
+
+We first go over required AWS Dependencies that are required to exist before we can run `helm install` in your EKS cluster.
+
+## AWS Dependencies 
+
+This section describes assumptions about existing AWS resources required run to the LLMEngine Server
+
+### EKS 
+The LLM Engine server must be deployed in an EKS cluster environment. Currently only versions `1.23+` are supported. Below are the assumed requirements for the EKS cluster: 
+
+One will need to provision EKS node groups with GPUs to schedule model pods. These node groups must have the `node-lifecycle: normal` label on them.
```

Comment:
nit: I think you can just say `You`, since we've already used that form earlier.